packages/graphql-language-service-interface/src/__tests__/getAutocompleteSuggestions-test.ts

```diff
@@ -476,4 +476,51 @@ query name {
       { label: 'Character' },
       { label: 'TestInterface' },
     ]));
+
+  it('provides correct types suggestions when typing a typename after a field', () => {
+    expect(testSuggestions('type Book { id: I', new Position(0, 18))).toEqual([
+      { label: '__DirectiveLocation' },
+      { label: '__TypeKind' },
+      { label: 'Episode' },
+      { label: 'InputType' },
```

Comment:
This is probably an existing bug, as the `InputType` should probably not appear here

Reply:
there's probably some simple logic we could add here for that if you want

![image](https://user-images.githubusercontent.com/1368727/104118342-4223b180-52f6-11eb-98dd-742a66f8cf67.png)

to access the GraphQLObjectType in this case, I think you want `state.prevState.type`. you should be able to getFields() and then match the fieldname up to check the type

Reply:
Thanks! But I don't seem to understand the role of field name here. 

In my current thinking, since `state.prevState.type`(type Book) is a GraphQLObjectType, and `InputType` is a GraphQLInputObjectType, so it should not appear in the hintList.

Reply:
my bad, misunderstood you before. i understand this bug better now. i’ll circle back after some sleep